RUN_NEW_TESTS.md

```
@@ -0,0 +1,59 @@
+# Quick Guide: Running the New Tests
+
+## Test Files Created
+
+1. **`tests/unit/test_workflow_validator.py`** - Unit tests for workflow validation
+2. **`tests/integration/test_simplified_workflows.py`** - Integration tests for workflow simplifications
+3. **`tests/integration/test_pr_agent_config_validation.py`** - Integration tests for configuration
+
+---
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)